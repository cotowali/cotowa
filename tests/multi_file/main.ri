require "./f.ri"
f_echo('hello f')
f2_echo('hello f2')
