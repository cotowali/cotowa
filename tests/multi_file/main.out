file: f
file: f2
echo_f
hello f
echo_f2
hello f2
